### DISCUSSION TITLE
Asking second question on different note.

### DISCUSSION THREAD

**Discussion by andsty** [logancyang/obsidian-copilot#550] [Q&A]
Hi. I am using llama3 using ollama. I ask question and mentioning [[note1]] and i get the correct answer. i ask a followup question but the response contains answer from question 1 and adds the correct answer for question2. i ask question 3 on [[note2]] and sometimes the reply will be some part from previous note answer and then it answers question 2. Is there a way to optimise it in order when i change notes in my question to get answer based on the note i am asking only?
Thanks in advance.

**Comment by logancyang** [comment]
It's partially because 1. the model is not tuned for RAG 2. you forgot to set a big context window explicitly in Ollama (check the local copilot guide).

Since this plugin focuses on customizability it loses some quality assurance if you don't have the best model combinations.

Personally I find GPT-4 level models work better in the current setup. That being said, this setup still has a lot to improve on for sure.
